src/theme/protocolTheme.ts

```diff
@@ -61,22 +63,24 @@ const getDesignTokens = (modeVal: ThemeMode) => ({
 });
 
 const useProtocolTheme = () => {
```

Comment:
@julienw because I am using `useAppDispatch()` here I am getting the following error
<img width="695" alt="Screenshot 2023-11-20 at 15 37 10" src="https://github.com/mozilla/perfcompare/assets/69891317/aa7c9d26-42ad-43da-a5b8-37656b9051c7">
Any idea on how to fix this?

Reply:
I wrote what I think is the solution in my comment above: you need to specify the `<Provider>` to `renderHook` using the `wrapper` argument in the `renderHook` cool. You'll also need to create a store. Maybe it's good to create a shared function `renderHook` in `src/__tests__/utils/test-utils.tsx` like we have for `render` already?